commit-status-publisher-server/src/main/java/jetbrains/buildServer/commitPublisher/github/ChangeStatusUpdater.java

```diff
@@ -243,15 +243,22 @@ private String getComment(@NotNull RepositoryVersion version,
               }
             }
 
-            if (artifacts != null && !artifacts.isEmpty()) {
-              comment.append("\n");
-              if (build.getBuildStatus() != Status.NORMAL) {
-                comment.append("Build status is not normal, not calculating artifact size\n");
-              } else {
-                comment.append(buildSize.getComment(build));
+            try {
+              String baseBranch = api.findBasePullRequestLabel(repositoryOwner,
+                                                               repositoryName, version.getVcsBranch());
+              if (baseBranch != null && baseBranch.equals(Constants.GITHUB_MASTER_BRANCH)) {
+                if (artifacts != null && !artifacts.isEmpty()) {
+                  comment.append("\n");
+                  if (build.getBuildStatus() != Status.NORMAL) {
+                    comment.append("Build status is not normal, not calculating artifact size\n");
+                  } else {
+                    comment.append(buildSize.getComment(build));
+                  }
+                }
               }
+            } catch (IOException e) {
+              LOG.warn("Failed to find base branch for " + version.getVcsBranch() + " for repository " + repositoryName);

```

Comment:
Please add this information into the comment field so when this happens it will be caught easily